src/actions/shoutouts.js

```diff
@@ -23,9 +25,26 @@ export const setError = error => {
 export const getSingleShoutout = id => async dispatch => {
   dispatch({ type: types.FETCHING_SHOUTOUT });
   try {
-    const res = await axiosWithAuth().get('/api/shoutouts/' + id)
+    const res = await axiosWithAuth().get('/api/shoutouts/' + id);
     dispatch(setSingleShoutout(res.data.data));
   } catch (err) {
     dispatch(setError(err.message));
   }
 };
+
+export const getProfileShoutouts = () => async dispatch => {
+  dispatch({ type: types.FETCHING_SHOUTOUT });
+  try {
+    const { data } = await Axios(localstorage.get().token).get(
```

Comment:
Good Job Man, Could you check out this Axios Instance, we fixed this on an earlier PR

Reply:
Resolved